packages/odie-client/src/data/use-send-odie-message.ts

```diff
@@ -42,9 +44,18 @@ export const useSendOdieMessage = () => {
 		odieBroadcastClientId,
 		setChatStatus,
 		setExperimentVariationName,
+		isUserEligibleForPaidSupport,
 	} = useOdieAssistantContext();
 
 	const addMessage = ( message: Message | Message[], props?: Partial< Chat > ) => {
+		if ( ! Array.isArray( message ) ) {
+			const isRequestingHumanSupport = message.context?.flags?.forward_to_human_support ?? false;
+			if ( isRequestingHumanSupport && isUserEligibleForPaidSupport ) {
+				newConversation();
+				return;
+			}
+		}
+
```

Comment:
Can we consider adding a Tracks Event here?
```suggestion
		isUserEligibleForPaidSupport,
		trackEvent,
	} = useOdieAssistantContext();

	const addMessage = ( message: Message | Message[], props?: Partial< Chat > ) => {
		if ( ! Array.isArray( message ) ) {
			const isRequestingHumanSupport = message.context?.flags?.forward_to_human_support ?? false;
			if ( isRequestingHumanSupport && isUserEligibleForPaidSupport ) {
				trackEvent( 'new_zendesk_chat' );
				newConversation();
				return;
			}
		}

```